### PR TITLE
Whisk: add preset files

### PR DESCRIPTION
**Pull request by dapplion**
Reduce shuffling sizes for minimal preset to have faster tests.

On offline discussions with @asn-d6 we noted that `CURDLEPROOFS_N_BLINDERS + WHISK_VALIDATORS_PER_SHUFFLE` must be a power of two, so for mainnet it's reduced from 128 to 124

### CODE REVIEW

File: presets/minimal/whisk.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Minimal preset - Whisk
+
+# Misc
+# ---------------------------------------------------------------
+# [customized]
+CURDLEPROOFS_N_BLINDERS: 2
```
Comment (asn-d6) [inline presets/minimal/whisk.yaml:6]: Curdleproofs always requires 4 blinders with the current state of the protocol.

Comment (dapplion) [reply presets/minimal/whisk.yaml:6]: Noting that https://github.com/nalinbhardwaj/curdleproofs.pie is able to validate a generated proof with those parameters

**Comment by asn-d6** [comment]
This looks good now! However it seems like merging this PR will cause a whole number of test breakage?

**Review by asn-d6 — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**